com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -920,15 +919,14 @@ protected virtual void Update()
                         // there are several bugs in this code, as we the message is dumped out under odd circumstances
                         //  For Matt, it would trigger when an object's rotation was perturbed by colliding with another
                         //  object vs. explicitly rotating it
-                        if (oldStateDirtyInfo.isPositionDirty || oldStateDirtyInfo.isScaleDirty || (oldStateDirtyInfo.isRotationDirty && SyncRotAngleX && SyncRotAngleY && SyncRotAngleZ))
+                        if (oldStateDirtyInfo.isPositionDirty || oldStateDirtyInfo.isScaleDirty || (oldStateDirtyInfo.isRotationDirty && (SyncRotAngleX || SyncRotAngleY || SyncRotAngleZ)))
```

Comment:
Interesting.  This code is better, for sure.  However, we can just remove the check for SyncRotAngle{XYZ}, because the code that makes it dirty only does so when the corresponding SyncRotAngle{XYZ} is true.